apps/cli/clixon_cli.h

```diff
@@ -60,3 +63,7 @@ int plugin_start(clicon_handle h, int argc, char **argv);
 int plugin_exit(clicon_handle h);
 
 #endif  /* _CLIXON_CLI_H_ */
+
+#ifdef __cplusplus
+} /* extern "C" */
+#endif
```

Comment:
for all files, better to have a new line at the end...
it may cause compilation warnings